.github/workflows/ci-testing.yml

```diff
@@ -115,7 +120,8 @@ jobs:
 
     - name: Tests
       env:
-          FIFTYONE_DO_NOT_TRACK: true
+        FLASH_TEST_TOPIC: ${{ matrix.topic }}
+        FIFTYONE_DO_NOT_TRACK: true
```

Comment:
shall t be rather just 0/1?

Reply:
Not sure if it can be. This was done by the fiftyone people for something on their end. I wouldn't want to change it in case something breaks.